core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferMemoryManager.java

```diff
@@ -155,6 +170,11 @@ public double getUtilization()
         return bufferedBytes.get() / (double) maxBufferedBytes;
     }
 
+    public TDigestHistogram getUtilizationHistogram()
```

Comment:
I would just return `TDigest` here and pack it into `TDigestHistogram` only when needed